packages/integration-react/src/MerkurComponent.jsx

```diff
@@ -191,26 +174,40 @@ export default class MerkurComponent extends React.Component {
 
     this._widget.unmount();
     this._widget = null;
+
+    if (this.props.onWidgetUnmounted) {
```

Comment:
I think that the onWidgetUnmounted method should get widget as argument and calling this function should be before calling widget.unmount. Have you got some examples where you need calling onWidgetUnmounted method after widget.unmount?

Reply:
`onWidgetUnmounted` has been renamed to `onWidgetUnmounting` and is called before the actual unmount call. Also it receives `this._widget` as an argument.